modulith/boot/src/main/java/io/edpn/backend/modulith/EDPNApplication.java

```diff
@@ -0,0 +1,12 @@
+package io.edpn.backend.modulith;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class EDPNApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(EDPNApplication.class, args);
```

Comment:
will this setup run the modulith module when running EDPNApplication ?

Reply:
yes, but since the build is so broken I've not gotten it to launch completely.